docs/reference/index.md

```diff
@@ -1,15 +1,16 @@
 <!--[metadata]>
 +++
-title = "Compose CLI reference"
+title = "Command line reference"
 description = "Compose CLI reference"
 keywords = ["fig, composition, compose, docker, orchestration, cli,  reference"]
 [menu.main]
 identifier = "smn_compose_cli"
-parent = "smn_compose_ref"
+parent = "workw_compose"
+weight=80
 +++
 <![end-metadata]-->
 
-## Compose CLI reference
+## Compose Command line reference
```

Comment:
Should "Command" be capitalized here? Probably doesn't matter too much, just looks a bit odd to me.